be/src/exprs/dictionary_get_expr.cpp

```diff
@@ -58,18 +60,24 @@ StatusOr<ColumnPtr> DictionaryGetExpr::evaluate_checked(ExprContext* context, Ch
     }
     value_chunk->reserve(size);
 
+    ColumnPtr null_column = UInt8Column::create(size, 0);
```

Comment:
Why reserve value_chunk, but resize null_column?

Reply:
Because probe_given_dictionary_cache maybe not change null_column if null_if_not_exist = false.